pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -1680,6 +1680,10 @@ type HTTPEndpointSpec struct {
 	// Indicate whether the server certificate is validated by the client or not
 	// +optional
 	DisableVerifySSL bool `json:"disableVerifySSL,omitempty"`
+	// Send the notifications with the CloudEvents header: https://github.com/cloudevents/spec/blob/main/cloudevents/adapters/aws-s3.md
+	// Supported for Ceph Quincy (v17) or newer.
```

Comment:
Is there a Ceph doc to link to as well? Questions I have and users might have (that hopefully Ceph docs clarify) are: 
- does enabling this decrease throughput?
- does enabling this disable something else?

Reply:
the ceph PR is not merged yet. related docs will be added to the bucket notification doc in ceph:
https://github.com/ceph/ceph/pull/44301/files#diff-3d118fb9c89f7fcd1a0b2c04df27367c6b7d8b788e52387a27af49820f28a328

this does not affect throughput or disable anything